internal/kubernetes/operator/features/crds.go

```diff
@@ -49,7 +51,7 @@ var (
 	ErrDocumentHasNoSpec         = errors.New("document contains no Spec")
 
 	versionsToResourcesMap = map[string][]resource{
-		"2.0.0": {
+		"2.1.0": {
 			resource{ResourceAtlasDatabaseUser, NopPatcher()},
 			resource{ResourceAtlasProject, NopPatcher()},
 			resource{ResourceAtlasDeployment, NopPatcher()},
```

Comment:
@s-urbaniak the nop patcher is so prominent that we might want to make it the default. So if nothing is set, that one is the one used. Will avoid a lot of changes that are not needed. We would only update when patching is required.